docs/core-concepts/authentication/sdk.mdx

```diff
@@ -1,230 +1,397 @@
 
 import Installation from '/snippets/sdk-installation.mdx';
 
-InsForge SDK provides complete authentication with JWT tokens, OAuth providers, and user profiles.
-
 <Installation />
 
-## Basic Authentication
+## signUp()
 
-### Sign Up
+Create a new user account with email and password.
 
-<CodeGroup>
-```javascript Basic Sign Up
-const { data, error } = await insforge.auth.signUp({
-  email: 'user@example.com',
-  password: 'secure_password123'
-})
+### Parameters
+
+- `email` (string, required) - User's email address
+- `password` (string, required) - User's password
+- `name` (string, optional) - User's display name
 
-// Returns: { data: { user, accessToken }, error }
+### Returns
+
+```typescript
+{
+  data: {
+    user: { id, email, name, emailVerified, createdAt, updatedAt },
+    accessToken: string
+  } | null,
+  error: Error | null
+}
 ```
 
-```javascript With Name
+### Example
+
+```javascript
 const { data, error } = await insforge.auth.signUp({
   email: 'user@example.com',
   password: 'secure_password123',
   name: 'John Doe'
 })
 ```
-</CodeGroup>
 
-### Sign In
+### Output
+
+```json
+{
+  "data": {
+    "user": {
+      "id": "usr_abc123",
+      "email": "user@example.com",
+      "name": "John Doe",
+      "emailVerified": false,
+      "createdAt": "2024-01-15T10:00:00Z",
+      "updatedAt": "2024-01-15T10:00:00Z"
+    },
+    "accessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."
+  },
+  "error": null
+}
+```
+
+---
+
+## signInWithPassword()
+
+Sign in an existing user with email and password.
+
+### Parameters
+
+- `email` (string, required) - User's email address
+- `password` (string, required) - User's password
+
+### Returns
+
+```typescript
+{
+  data: {
+    user: { id, email, name, emailVerified, createdAt, updatedAt },
+    accessToken: string
+  } | null,
+  error: Error | null
+}
+```
+
+### Example
 
 ```javascript
 const { data, error } = await insforge.auth.signInWithPassword({
   email: 'user@example.com',
   password: 'secure_password123'
 })
+```
 
-if (data) {
-  // Token is automatically stored and used for future requests
-  console.log('Logged in:', data.user)
-  console.log('Token:', data.accessToken)
+### Output
+
+```json
+{
+  "data": {
+    "user": {
+      "id": "usr_abc123",
+      "email": "user@example.com",
+      "name": "John Doe",
+      "emailVerified": true,
+      "createdAt": "2024-01-15T10:00:00Z",
+      "updatedAt": "2024-01-15T10:00:00Z"
+    },
+    "accessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."
+  },
+  "error": null
 }
 ```
 
-### Sign Out
+---
 
-```javascript
-const { error } = await insforge.auth.signOut()
-// Clears stored tokens and session
+## signInWithOAuth()
+
+Start OAuth authentication flow with Google or GitHub.
+
+### Parameters
+
+- `provider` ('google' | 'github', required) - OAuth provider
+- `redirectTo` (string, optional) - URL to redirect after authentication
+- `skipBrowserRedirect` (boolean, optional) - Return URL instead of auto-redirecting
```

Comment:
if skipBrowserRedirect=true, what will happen and how to handle the result, we should make them clear.